pkg/config/config.go

```diff
@@ -97,22 +97,24 @@ func (c *Config) watch() {
 	configFiles := c.configFiles()
 	err := e.configFilesChangedObserver.Watch(configFiles)
 	if err != nil {
-		glog.Errorf("watch [Error]: %v", err)
+		glog.Error("Can't watch configuration files: %s", err)
+		return
+	}
+	for _, file := range configFiles {
+		glog.Infof("Watching configuration file '%s'", file)
 	}
-	glog.Infof("watch: %v", configFiles)
 
 	// Load new configuration when config files change.
 	e.configFilesChangedObserver.AddListener(func(event interface{}) {
```

Comment:
if we do not use the event var, we can use some shorter name, or _ ?

Reply:
Yes, sure, I will change that.